src/cli/util.js

```diff
@@ -448,6 +449,16 @@ function formatFiles(context) {
     context.logger.log("Checking formatting...");
   }
 
+  let changedCache = null;
+  if (context.argv["only-changed"]) {
+    changedCache = new ChangedCache(
+      fs,
+      process.env.PRETTIER_CACHE_LOCATION || ".prettiercache",
```

Comment:
I think better use `https://www.npmjs.com/package/find-cache-dir` for location

Reply:
I considered that option, but it looked [https://github.com/avajs/find-cache-dir/issues/1](controversial) so I opted to use a different approach.

Reply:
/cc @prettier/core

Reply:
hm, 99% tools use `node_modules/.cache`, only `eslint` doesn't do this, i think better store cache as do other tools, because it is require many developers change cache location and it is bad DX